modules/AWS/eks/main.tf

```diff
@@ -12,8 +12,9 @@ module "eks" {
   enable_irsa = true
 
   # Networking
-  vpc_id  = var.vpc_id
-  subnets = var.subnets
+  vpc_id                    = var.vpc_id
+  subnets                   = var.subnets
+  cluster_service_ipv4_cidr = "172.20.0.0/16"
```

Comment:
Restricting IPv4 range for every serviceIP in K8S cluster

Reply:
Please add this in local to avoid the literal in code.